pytroll_collectors/segments.py

```diff
@@ -191,6 +193,24 @@ def _adjust_time_by_flooring(self, metadata, group_by_minutes, time_name):
                                 time_item.day, time_item.hour, floor_minutes, 0)
         metadata[time_name] = time_item
 
+    def _check_scheme(self, posttroll_message):
```

Comment:
just a naming comment here, how about `_handle_scheme` instead? For me it conveys better that the message might be changed.